app/views/home/index.html.haml

```diff
@@ -122,7 +122,7 @@
         .home-card.home-twitter-news
           %a.twitter-timeline{"data-height" => "360", :href => "https://twitter.com/lagroportal?ref_src=twsrc%5Etfw"}
             .home-twitter-loader 
-              = render LoaderComponent.new
+              = render LoaderComponent.new(type: '2')
```

Comment:
`type: '2'` is not self explanatory, replace with `= render LoaderComponent.new(type: 'ilips')`